frontend/src/app/organization/organization-contact/organization-contact.component.ts

```diff
@@ -0,0 +1,30 @@
+import {Component, EventEmitter, Input, OnInit, Output, ViewChild} from '@angular/core';
+import { NgForm } from '@angular/forms';
+import { OrganizationContactPayload } from '../organization.model';
+
+@Component({
+  selector: 'volontulo-organization-contact',
+  templateUrl: './organization-contact.component.html',
+  styleUrls: ['./organization-contact.component.css']
+})
+export class OrganizationContactComponent implements OnInit {
+  @ViewChild('contactForm') contactForm: NgForm;
```

Comment:
Answering @magul concern: `ViewChild` is okay in this use case. However, you might just pass form value to the `onSubmit` method in the template, which would allow to avoid this `ViewChild` here.